frontend/src/components/associations/library/management/loans/LoanStatusBadge.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Loan } from "../../../../../models/associations/library";
+import Badge from "react-bootstrap/Badge";
+
+export const LoanStatusBadge = ({ status }: Pick<Loan, "status">) => {
+  switch (status) {
+    case "ACCEPTED":
+      return <Badge variant="primary">Accepté</Badge>;
```

Comment:
Take advantage of the `Tag` component to be visually consistent with marketplace